oak_runtime/introspection_browser_client/npm_hook_package_added_or_removed

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Removes the stale ./node_modules/.cache/last-install-package-lock.json after
+# an NPM package has been added or removed. This additional hook is needed since
+# that file can only be genereated after dependencies are installed as whole,
+# but not when individual dependencies are added or removed.
+
+# Check if this hooks was called from a dependency, not from the source package.
+if [[ $INIT_CWD != $PATH ]]; then
```

Comment:
where is this `INIT_CWD` variable coming from? Could you link to some documentation / reference?